CHANGELOG.md

```diff
@@ -10,12 +10,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added --pki_prod option that can be forwarded to Speculos upon usage
+## [1.28.0] - 2025-03-04
+
+### Changed
+
+- Use --Verbose option to increase verbosity. Default behavior is to remove timestamps and useless logs. The option is forwarded to speculos.
```

Comment:
If it's a default behavior it's hard to say it removes anything because we won't know to what it's compared to.  

```suggestion
- Use `--Verbose` option to increase `pytest` verbosity. It add log timestamps, some test-specific logs, and the verbose option is forwarded to Speculos.
```